.circleci/config.yml

```diff
@@ -133,25 +133,100 @@ jobs:
           command: |
             gsutil -m rsync -r ~/embeddable-explorer/dist gs://embeddable-explorer/$CIRCLE_SHA1
 
+  build-latest-umd-sandbox:
+    executor: node
+    steps:
+      - checkout
+      - npm-install
+      - run:
+          name: Build
+          command: '[ -d build ] || PUBLIC_URL=https://embeddable-explorer.cdn.apollographql.com/_latest npm run build-explorer:umd -- --no-progress'
+          environment:
+            BUILD_TO_CDN: 'true'
+      - persist_to_workspace:
+          root: ~/embeddable-explorer
+          paths:
+            - dist
+
+  build-versioned-umd-sandbox:
+    executor: node
+    steps:
+      - checkout
+      - npm-install
+      - run:
+          name: Build
+          command: '[ -d build ] || PUBLIC_URL=https://embeddable-sandbox.cdn.apollographql.com/$CIRCLE_SHA1 npm run build-sandbox:umd -- --no-progress'
+          environment:
+            BUILD_TO_CDN: 'true'
+      - persist_to_workspace:
+          root: ~/embeddable-explorer
+          paths:
+            - dist
+
+  cdn-upload-latest-sandbox:
+    executor: gcp-cli/google
+    steps:
+      - gcp-cli/initialize:
+          gcloud-service-key: GCLOUD_GCS_SERVICE_KEY
+      - gcp-cli/install
+      - checkout
+      - attach_workspace:
+          at: ~/embeddable-explorer
+      - run:
+          name: Upload build artifacts to CDN
+          command: |
+            gsutil -m rsync -r ~/embeddable-sandbox/dist gs://embeddable-sandbox/_latest
```

Comment:
@cheapsteak I created a new bucket for these files: https://console.cloud.google.com/storage/browser/embeddable-sandbox;tab=objects?project=mdg-services&prefix=&forceOnObjectsSortingFiltering=false

Just an fyi